dev-libs/libgee/patches/libgee-0.20.1.patchset

```diff
@@ -0,0 +1,29 @@
+From 4e75be78feb57dddf26f7ef91cef1e0d0d25a672 Mon Sep 17 00:00:00 2001
+From: begasus <begasus@gmail.com>
+Date: Thu, 3 Jan 2019 16:04:15 +0100
+Subject: set correct path to install the gobject-introspection files
+
+
+diff --git a/gee/Makefile.am b/gee/Makefile.am
+index 42e0a89..909b09d 100644
+--- a/gee/Makefile.am
++++ b/gee/Makefile.am
+@@ -147,13 +147,13 @@ dist_vapi_DATA = \
+ 	$(NULL)
+ 
+ if HAVE_INTROSPECTION
+-girdir = @INTROSPECTION_GIRDIR@
++girdir = @datadir@/gir-1.0
```

Comment:
I think this can be automated with `sed` instead.

Reply:
Need to look into this, but yes could be applied with sed I guess :)